sources/osg/Program.js

```diff
@@ -82,6 +94,14 @@ Program.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( GLO
         return new Program();
     },
 
+    dirty: function () {
```

Comment:
Why only program and not StateAttribute get the dirty() and isDirty() accessor ?